extensions/stats/plugin.cc

```diff
@@ -87,13 +87,18 @@ void PluginRootContext::report(
       CTXDEBUG("metricKey cache hit ", istio_dimensions_.debug_key(),
                ", stat=", stat.metric_id_, stats_it->first.to_string());
     }
-    incrementMetric(cache_hits_, 1);
+    cache_hits_accumulator_++;
+    if (cache_hits_accumulator_ == 100) {
+      incrementMetric(cache_hits_, cache_hits_accumulator_);
```

Comment:
Why accumulate? Is it expensive?

Reply:
Yes, this is adding one metric call into the host for every report. We do not need that level of fidelity. updating every 100 calls is ok.